kaldb/pom.xml

```diff
@@ -567,6 +566,10 @@
                     <artifactId>maven-clean-plugin</artifactId>
                     <version>2.5</version>
                 </plugin>
+                <plugin>
+                    <artifactId>maven-deploy-plugin</artifactId>
+                    <version>2.8.1</version>
```

Comment:
We probably should set this to `3.0.0-M1`

Reply:
Done.